packages/applications-service-api/Dockerfile

```diff
@@ -12,15 +12,16 @@ ADD api api
 ADD src src
 ADD __tests__ __tests__
 ADD jest.config.js jest.config.js
+ADD .eslintrc .eslintrc
 ADD data data
 
 ADD package.json package.json
 ADD .sequelizerc .sequelizerc
 ADD package-lock.json package-lock.json
 
-RUN npm i
+RUN npm i --production
```

Comment:
I think this line should be just `npm i` since we need dev dependencies for the `npm run documetation` step

Reply:
done